x-pack/plugin/watcher/src/main/java/org/elasticsearch/xpack/watcher/notification/email/EmailService.java

```diff
@@ -104,7 +104,11 @@
     private final CryptoService cryptoService;
 
     public EmailService(Settings settings, @Nullable CryptoService cryptoService, ClusterSettings clusterSettings) {
-        super("email", clusterSettings, EmailService.getSettings());
+        super("email", clusterSettings, Arrays.asList(SETTING_DEFAULT_ACCOUNT, SETTING_PROFILE, SETTING_EMAIL_DEFAULTS, SETTING_SMTP_AUTH,
```

Comment:
this is a lot of duplication from the setting. If a new settings gets added it needs to be added here as well. Would it make sense to have `EmailService.getSettings()` and `EmailService.getSecureSettings()` for each of the service? This also applies for the next changes